TypeScript/ts-jest-test-setup/index.ts

```diff
@@ -0,0 +1,7 @@
+export const testFunction = (names : string[]) => {
```

Comment:
Shouldn't this function have a return value? I guess in this case it would be string | number? Same in index.ts mocha test setup

Reply:
The comments on line 5-6 should work?